File: LogicadeProgramacao/Estrutura de Decisão/2. Java/ReajusteSalarial.java
import java.util.Scanner;

public class ReajusteSalarial {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Digite o salário atual do colaborador: ");
        double salario = input.nextDouble();
        double novoSalario, aumento;

        if (salario <= 280) {
            aumento = salario * 0.2;
            novoSalario = salario + aumento;
            System.out.printf("Salário antes do reajuste: R$ %.2f%n", salario);
            System.out.println("Percentual de aumento aplicado: 20%");
        } else if (salario <= 700) {
            aumento = salario * 0.15;
            novoSalario = salario + aumento;
            System.out.printf("Salário antes do reajuste: R$ %.2f%n", salario);
            System.out.println("Percentual de aumento aplicado: 15%");
        } else if (salario <= 1500) {
            aumento = salario * 0.1;
            novoSalario = salario + aumento;
            System.out.printf("Salário antes do reajuste: R$ %.2f%n", salario);
            System.out.println("Percentual de aumento aplicado: 10%");
        } else {
            aumento = salario * 0.05;
            novoSalario = salario + aumento;
            System.out.printf("Salário antes do reajuste: R$ %.2f%n", salario);
            System.out.println("Percentual de aumento aplicado: 5%");
        }

        System.out.printf("Valor do aumento: R$ %.2f%n", aumento);
        System.out.printf("Novo salário: R$ %.2f%n", novoSalario);
        }
}

File: LogicadeProgramacao/Estrutura de Decisão/3. Python/ReajusteSalarial.py
salario = float(input("Digite o salário atual do colaborador: "))
novo_salario = 0
aumento = 0

if salario <= 280:
    aumento = salario * 0.2
    novo_salario = salario + aumento
    print(f"Salário antes do reajuste: R$ {salario:.2f}")
    print("Percentual de aumento aplicado: 20%")
    
elif salario <= 700:
    aumento = salario * 0.15
    novo_salario = salario + aumento
    print(f"Salário antes do reajuste: R$ {salario:.2f}")
    print("Percentual de aumento aplicado: 15%")
    
elif salario <= 1500:
    aumento = salario * 0.1
    novo_salario = salario + aumento
    print(f"Salário antes do reajuste: R$ {salario:.2f}")
    print("Percentual de aumento aplicado: 10%")
    
else:
    aumento = salario * 0.05
    novo_salario = salario + aumento
    print(f"Salário antes do reajuste: R$ {salario:.2f}")
    print("Percentual de aumento aplicado: 5%")

print(f"Valor do aumento: R$ {aumento:.2f}")
print(f"Novo salário: R$ {novo_salario:.2f}")